pkgs/top-level/all-packages.nix

```diff
@@ -11742,7 +11742,7 @@ with pkgs;
   rmtrash = callPackage ../tools/misc/rmtrash { };
 
   roc-toolkit = callPackage ../development/libraries/audio/roc-toolkit {
-    scons = sconsPackages.scons_4_1_0;
+    scons = __splicedPackages.sconsPackages.scons_4_1_0;
```

Comment:
The override has been removed in staging.

Reply:
thanks